eos/b-decays/b-to-d-l-nu.cc

```diff
@@ -80,7 +80,58 @@ namespace eos
             u.uses(*model);
         }
 
-        // normalized to V_cb = 1
+        //* normalized(|Vcb|=1) two-fold-distribution, cf. [DSD2014], eq. (13), p. 6
+        double normalized_two_differential_decay_width(const double & s, const double & c_theta_l) const
+        {
+            //  d^2 Gamma, cf. [DSD2014], p. 6, eq. (13)
+            // Trigonometric identities
+            double c_theta_l_2 = c_theta_l * c_theta_l;
```

Comment:
From here on to line 124 is code used multiple times in these three functions 

normalized_two_differential_decay_width
normalized_differential_decay_width
numerator_differential_a_fb_leptonic

Please create some struct that calculates these quantities for "s" and
stores them, so you can call it and use them whenever you calculate
an observable - much like in B->D* l v

Reply:
You did not remove the code duplication?!

Reply:
i tried something for this but it didn't work.., so i prefer to keep it this way, unless you want to do this before merging.

Reply:
Should'nt be similar to B->D^* lv where you have the struct of the amplitudes?